public/src/js/widgets/columns/fronts-config.html

```diff
@@ -264,6 +264,16 @@
             <input type="checkbox" data-bind="
                 checked: meta.uneditable" />
 
+            <div class="platform-edit">
+                <span>Platform: </span><span data-bind="text: getPlatform()"></span>
+                <!-- ko if: !thisIsPlatformSpecificCollection() && parents().length <= 1 -->
```

Comment:
Do the comments need to stay?

Reply:
this is actually how you conditionally create html in knockout. It's great!

Reply:
Fun (hopefully on react's roadmap to introduce this feature)